openupgrade_scripts/scripts/account_check_printing/15.0.1.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+from openupgradelib import openupgrade
+
+
+def _create_account_payment_method_line(env):
```

Comment:
Can you please express the logic behind adding these lines?

Reply:
The key is in the line `WHERE apm.code = 'check_printing'`. The payment methods of this code need corresponding payment method lines.

Reply:
That's still no reason. Please indicate the code where that lines are needed, and why only for checks.

Reply:
In `account` module, we have created the ones for:
![Selection_722](https://user-images.githubusercontent.com/25005517/212649086-87d1219d-c4e8-47a0-882e-2146e6e3976e.png)
In `payment` module, we have created the ones for:
![Selection_721](https://user-images.githubusercontent.com/25005517/212649465-b49b9ff0-87c7-4820-b0da-2293ad6cc63e.png)
So it's logical we need to populate in this PR the ones for `code = 'check_printing'`.
About why we need to create payment method lines for the payment methods... well, it's just the new methodology, we need records in this new model in order to make all this the payment method logic be functional. I mean, the payment methods in this account_check_printing module uses this code https://github.com/odoo/odoo/blob/14.0/addons/account_check_printing/models/account_journal.py#L69.

Reply:
OK, that's what the reference I was looking for. Please include it as comment in the code and then I'll merge it.

Reply:
Done

Reply:
I don't see the code link in the comment...

Reply:
Added code link. The problem is that it was too long for flake8 tastes, but well, I put a shortened version.

Reply:
What do you mean about "`(...)/account_check_printing/models/account_journal.py#L69`" doesn't serve? I don't understand you. It's super-pretty-obvious that `(...)` means "`https://github.com/odoo/odoo/tree/15.0/addons/`".

Reply:
Ah, and how I obtain that permalink?

Reply:
![Selección_014](https://user-images.githubusercontent.com/7165771/212718412-8095f231-148e-4132-b5dd-99d8af4473fc.png)

Reply:
Thanks. Ready!